tests/virt_autotest/reboot_and_wait_up_normal.pm

```diff
@@ -16,9 +16,23 @@ use testapi;
 use base "reboot_and_wait_up";
 
 sub run {
+    if (check_var('ARCH', 's390x')) {
+        #type reboot
+        my $cmd = "reboot -f\n";
+        type_string "$cmd";
+
+        #Wait for s390x lpar bootup
+        sleep 120;
+
+        #Switch to s390x lpar console
+        reset_consoles;
+        my $svirt               = select_console('svirt', await_console => 0);
+    }
+    else {
     my $self    = shift;
     my $timeout = 180;
     $self->reboot_and_wait_up($timeout);
```

Comment:
as above

Reply:
Had made code changed as your mentioned, after local verification run, will update this PR,ASAP.

Reply:
@okurz move new codes into reboot_and_wait_up as basically, updated this PR and
Verification Run:
http://10.67.17.251/tests/90